README.md

```diff
@@ -214,6 +214,21 @@ The cypress-report.html file is generated and should be in (/integration-tests/s
 
 ## References
 
-- [Console Plugin SDK README](https://github.com/openshift/console/tree/master/frontend/packages/console-dynamic-plugin-sdk)
-- [Customization Plugin Example](https://github.com/spadgett/console-customization-plugin)
-- [Dynamic Plugin Enhancement Proposal](https://github.com/openshift/enhancements/blob/master/enhancements/console/dynamic-plugins.md)
+To learn more, have a look at the SDK documentation, a few sample plugins, and
```

Comment:
```suggestion
To learn more, have a look at the SDK documentation, a few sample plugins, and
```
I am wondering if it might just be better to say something like "To learn more, refer to the following links" or "For more information, see the plugin documentation, sample plugins, and the source code for a few active plugins." to be more consistent with the product documentation.